stacks/api/bucket/src/utility.ts

```diff
@@ -170,3 +172,72 @@ export function buildRelationAggregation(
     ];
   }
 }
+
+export function provideLanguageChangeUpdater(
+  bucketService: BucketService,
+  bucketDataService: BucketDataService
+) {
+  return (previousSchema: object, currentSchema: object) => {
```

Comment:
How about we make the returned function an async function? I believe if we get rid of .then chaining, it'd be more readable.

This is only a suggestion. You might like it this way. NP